docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -155,42 +155,43 @@ For more details on getting started with the Synthetics Recorder, refer to <<syn
 == Playwright syntax
 
 Inside the callback for each step, you'll likely use a lot of Playwright syntax.
-Many Playwright classes are supported in the Elastic Synthetics library to simulate
-user workflows including tasks like:
+Use Playwright to simulate and validate user workflows including:
 
 * Interacting with the https://playwright.dev/docs/api/class-browser[browser]
   or the current https://playwright.dev/docs/api/class-page[page] (like in the example above).
 * Finding elements on a web page using https://playwright.dev/docs/api/class-locator[locators].
 * Simulating https://playwright.dev/docs/api/class-mouse[mouse],
   https://playwright.dev/docs/api/class-touchscreen[touch], or
   https://playwright.dev/docs/api/class-keyboard[keyboard] events.
+* Making assertions using https://playwright.dev/docs/test-assertions[`@playwright/test`'s `expect` function]. Read more in <<synthetics-make-assertions>>.
 
 Visit the https://playwright.dev/docs[Playwright documentation] for information.
 
 However, not all Playwright functionality should be used with Elastic Synthetics.
-
 In some cases, there are alternatives to Playwright functionality built into the
 Elastic Synthetics library. These alternatives are designed to work better for
 synthetic monitoring. Do _not_ use Playwright syntax to:
 
-* *Make assertions*. Use Elastic Synthetics's `expect` methods instead.
-  Read more in <<synthetics-make-assertions>>.
 * *Make API requests.* Use Elastic Synthetic's `request`
   parameter instead. Read more in <<synthetics-request-param>>.
 
 There is also some Playwright functionality that is not supported out-of-the-box
 in Elastic Synthetics including:
 
 * https://playwright.dev/docs/api/class-video[Videos]
+* The https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-screenshot-1[`toHaveScreenshot`] and https://playwright.dev/docs/api/class-snapshotassertions[`toMatchSnapshot`] assertions
 
 [discrete]
 [[synthetics-make-assertions]]
 = Make assertions
 
-Check that a value meets a specific condition.
 A more complex `step` might wait for a page element to be selected
 and then make sure that it matches an expected value.
 
+Elastic Synthetics uses `@playwright/test`'s `expect` function to make assertions and
```

Comment:
Would call this unsupported one as snapshot based assertions.  `except for the unsupported ones [Snapshot Assertions](https://playwright.dev/docs/api/class-snapshotassertions) toHaveScreenshot and toMatchSnapsho`